### PR TITLE
Add number places to jsp

### PR DESCRIPTION
**Pull request by OlgaKochneva**
1. Fix calculating place number
2. Add logic to price calculating depend on places number
3. Add places number to jsp
4. Fix dates in frontend

### CODE REVIEW

**Comment by OlgaKochneva** [comment]
[[BE] add number free places on jsp + fix that number calculating](https://trello.com/c/uqIh4bgQ/84-be-add-number-free-places-on-jsp-fix-that-number-calculating)

File: src/main/java/com/epam/lowcost/DAO/implementations/TicketDAOImpl.java [hunk 1]
```diff
@@ -93,26 +93,17 @@ public boolean deleteTicketsByUserId(long id) {
...
-            JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
-            List<Ticket> tickets = jdbcTemplate.query(joinQuery() + "and flightId=? and isBusiness=?",
-                    rowMapper, id, false);
-            n = tickets.size();
+            n = jdbcTemplate.queryForObject("SELECT COUNT (*) FROM TICKETS " +
+                            "JOIN  USERS ON TICKETS.userId=USERS.id " +
+                            "JOIN  FLIGHTS ON TICKETS.flightId=FLIGHTS.id " +
+                            "JOIN  PLANES ON FLIGHTS.planeId = PLANES.id " +
+                            "WHERE TICKETS.isDeleted=false and flightId=? and isBusiness=?", Integer.class,
+                    flightId, isBusiness);
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/DAO/implementations/TicketDAOImpl.java:112]: u need to business places, why u don't use `isBusinness = false` ?

File: src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java [hunk 2]
```diff
@@ -89,20 +92,48 @@ private long calculateInitialFlightPriceByDate(long daysBetween, long minPrice)
...
         long daysBetween = DAYS.between(dateBefore, dateAfter);
         long minPrice = flight.getInitialPrice();
-        flight.setInitialPrice(calculateInitialFlightPriceByDate(daysBetween, minPrice));
+        long decreaseBusinessPlacesIncreasePrice = 0;
+        long decreaseEconomyPlacesIncreasePrice = 0;
+        long decreaseDaysBetweenIncreasePrice = calculateInitialFlightPriceByDate(daysBetween, minPrice);
+        // if number of free business places less than quarter of plane business places capacity
+        // price rises on quarter of min price
+        if (getNumberOfFreeBusinessPlaces(flight) < flight.getPlane().getBusinessPlacesNumber() / 4.0) {
+            decreaseBusinessPlacesIncreasePrice = (long) (minPrice / 4.0);
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java]: All constants should be moved into separate variables

File: src/main/webapp/WEB-INF/views/users.jsp [hunk 3]
```diff
@@ -84,33 +87,33 @@
...
-    <%--<input type="email" name="email"/> <spring:message code="lang.email"/><br/>--%>
-    <%--<input type="password" name="password"/> <spring:message code="lang.password"/><br/>--%>
-    <%--<input type="text" name="isAdmin"/> <spring:message code="lang.isAdmin"/><br/>--%>
-    <%--<input type="text" name="firstName"/> <spring:message code="lang.firstName"/> <br/>--%>
-    <%--<input type="text" name="lastName"/> <spring:message code="lang.lastName"/> <br/>--%>
-    <%--<input type="text" name="documentInfo"/> <spring:message code="lang.document"/> <br/>--%>
-    <%--<input type="datetime-local" name="birthday"/> <spring:message code="lang.birthday"/> <br/>--%>
-    <%--<input type="submit" value="OK"/>--%>
+<%--<input type="email" name="email"/> <spring:message code="lang.email"/><br/>--%>
+<%--<input type="password" name="password"/> <spring:message code="lang.password"/><br/>--%>
```
Comment (vakhonin) [inline src/main/webapp/WEB-INF/views/users.jsp]: removed all comments, why you save them ?

Comment (OlgaKochneva) [reply src/main/webapp/WEB-INF/views/users.jsp]: I deleted

File: src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java (hunk 2 above)
Comment (OlgaKochneva) [reply src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java]: done

**Pull request merged**